src/QueryM.purs

```diff
@@ -252,6 +254,15 @@ getChainTip = ogmiosChainTipToTip <$> mkOgmiosRequest Ogmios.queryChainTipCall
     Ogmios.CtChainPoint { slot, hash } -> Chain.Tip $ wrap
       { slot, blockHeaderHash: wrap $ unwrap hash }
 
+waitUntilSlot :: AbsSlot -> QueryM Chain.Tip
```

Comment:
Can we have more commentary on this function?

Reply:
Also it could contain a note about the current excessive querying so that we don't forget about it eventually